sdk/core/Azure.Core/src/OperationOfT.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure
```

Comment:
Will customers ever use this type directly?  Or just our devs when providing implementations of it?  Should it be in `Azure` or `Azure.Core`?

Reply:
LORs that don't have complicated state will return Operation<T>, so it will be as used by users as Response<T>.